src/adaptation/massmatrix.jl

```diff
@@ -112,10 +112,32 @@ function Base.resize!(wv::WelfordVar, θ::AbstractVecOrMat{T}) where {T<:Abstrac
     end
 end
 
+function Base.resize!(wv::WelfordVar, θ::AbstractVector{T}) where {T<:AbstractFloat}
+    if length(θ) != size(wv.var, 1)
+        @assert wv.n == 0 "Cannot resize a var estimator when it contains samples."
+        n = length(θ)
+        __fill!(wv.μ, n, T(0))
+        __fill!(wv.M, n, T(0))
+        __fill!(wv.δ, n, T(0))
+        __fill!(wv.var, n, T(1))
+    end
+end
+
+function __fill!(x::AbstractVector, n, V)
```

Comment:
For clarity: 

```suggestion
function __resize_fill!(x::AbstractVector, n, V)
```

Also, please add a docstring to this function.